web/assets/css/bundle_card.css

```diff
@@ -0,0 +1,151 @@
+.bundle-card-line {
+    overflow: hidden;
+    margin-top: 20px;
+}
+
+.bundle-card-line-data {
+    background: rgb(241, 241, 241);
+    overflow: hidden;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    padding: 20px 5px;
+}
+
+.bundle-card-line-content h4 {
+    font-weight: 700;
+}
+
+.bundle-card-line-author-username {
+    background: rgb(231,231,231);
+    width: 100%;
+    padding: 15px 0;
+    text-align: center;
+    font-weight: 700;
+    font-size: 16px;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+}
+
+.bundle-card-line-author-photo {
+    padding: 15px 25px;
+    background: rgb(255,255,255);
+    text-align: center;
+}
+
+.bundle-card-line-last-update {
+    background: rgb(231,231,231);
+    width: 100%;
+    padding: 9px 0;
+    text-align: center;
+    font-size: 16px;
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+
+.bundle-card-line-last-update span {
+    display: block;
+    font-weight: 700;
+}
+
+.bundle-card-line-contributors {
+    background: rgb(217, 217, 217);
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+    padding: 20px 30px;
+}
+
+
+.bundle-card-line-contributors-container {
+    overflow: hidden;
+}
+
+.bundle-card-line-contributors span {
+    display: block;
+    float: left;
+    font-size: 18px;
+    padding-top: 10px;
+}
+
+.bundle-card-line-contributor {
+    background: rgb(255,255,255);
+    float: left;
+    margin-left: 20px;
+    border-radius: 5px;
+    padding: 3px;
+}
+
+.bundle-card-line-contributor .bundle-card-line-contributor-photo {
+    float: left;
+    overflow: hidden;
+    border-radius: 5px;
+}
+
+.bundle-card-line-contributor .bundle-card-line-contributor-username {
+    float: left;
+    overflow: hidden;
+    padding: 10px 10px;
+}
+
+.bundle-card-line-contributors-more {
+    margin-top: 10px;
+    margin-left: 45px;
+}
+
+@media (max-width: 768px) {
+    .bundle-card-line {
+        margin-top: 10px;
+    }
+
+    .bundle-card-line-data {
+        padding: 10px 0;
+        border-radius: 5px;
+    }
+
+    .bundle-card-line-author {
+        overflow: hidden;
+    }
+
+    .bundle-card-line-content h4 {
```

Comment:
Change to h2 (ref: https://github.com/ezsystems/ezplatform-com/pull/80/files#diff-205461f0c399ee7cdb4ed7a7eda49f76R4)